mwc-select not present in npm registry

```
npm i @material/mwc-select

npm ERR! code E404
npm ERR! 404 Not Found: @material/mwc-select@latest

npm ERR! A complete log of this run can be found in:
npm ERR!     .npm/_logs/2018-05-14T09_06_41_487Z-debug.log
zsh: exit 1     npm i @material/mwc-select
```

Can you please add it to npm?

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # Material Web Components
 
-> :warning: These components are still a work in progress. Not all elements are available on NPM yet. We are working on the remaining elements and publish them to registry once we have a first working version. :warning:
+> :warning: These components are still a work in progress. Not all elements are available on NPM yet. We are working on the remaining elements and publish them to the NPM registry once we have a first working version. :warning:
 
 Material Web Components helps developers execute [Material Design](https://www.material.io) using [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
 

